src/components/accessories/langSwitcher/LangSwitcher.tsx

```diff
@@ -2,38 +2,25 @@ import React, { FunctionComponent, useContext } from "react";
 import i18n from "i18next";
 import { LangContext } from "../../../libraries/langContext/langContext";
 import "./styles.scss";
-
-const languageVocabolary: Record<string, string> = {
-  en: "English",
-  it: "Italian",
-  de: "German",
-  fr: "French",
-  es: "Spanish",
-  pt: "Portoguese",
-  ar: "Arabic",
-  sw: "Swahili",
-  am: "Amharic",
-  cs: "Czech",
-  sq: "Albanian",
-  zh: "Chinese",
-};
```

Comment:
I would leave this as fallback in case that the available languages inside the customisation file was empty.

Please handle this and then we can merge the code.